### ISSUE TITLE
Translate CLI content

### ISSUE THREAD

**Issue by Baspa** [vormkracht10/laravel-seo-scanner#48]
Only the failure reason is translated for each check, while the check name and other content is not translated. We could translate those as well. We might need help with the French translations.

**Comment by Baspa** [comment]
@ThibaultDuhem would you like to help me when I'm adding more translations to the translation files? Not all text is currently translated and I would like to add those to the translation files as well.

**Comment by ThibaultDuhem** [comment]
@Baspa Yes, of course. I'll be happy to assist you with the new translation keys in the french translation file. Just mention me when new keys are available in English, and I'll add them to the French translation file.

**Comment by MuriloChianfa** [comment]
Hello guys, im testing the package and i've noticed that's language not working for me:

![image](https://github.com/user-attachments/assets/546e58da-b152-4afb-8383-2e284d354254)

I had tested tested using language => null and language => 'en', but not work.

**Comment by MuriloChianfa** [comment]
im using the latest version too, but the translations not working :(

**Comment by Baspa** [comment]
Did you also set the locale in your .env file? @MuriloChianfa

**Comment by MuriloChianfa** [comment]
Testing again here, setting the config app.locale to "en" it works successfully, but my default app.locale is "en_US", that's why it didn't work.

So the config seo.language does not working?
![image](https://github.com/user-attachments/assets/c1fa1f92-064b-46fe-b479-8e3918f9efc5)

**Comment by Baspa** [comment]
Hmm I would expect that the language from the seo config would overrule the app config but maybe that's broken. Have to check that to make sure that isn't working. @MuriloChianfa

**Comment by Baspa** [comment]
Ah, forgot what the language was used for but I checked it and the `language` option in the config does only work for the `TransitionWordRatioCheck`. It checks on transition words on the page and to do so it needs to know the language of the website. @MuriloChianfa 

So to change the locale of the output of the scan you should use the normal `app` config.
